apps/src/applab/applab.js

```diff
@@ -680,7 +694,7 @@ Applab.init = function(config) {
     // should never be present on such levels, however some levels do
     // have levelHtml stored due to a previous bug. HTML set by levelbuilder
     // is stored in startHtml, not levelHtml.
-    if (config.level.hideDesignMode) {
+    if (Applab.reduxStore.getState().isDesignModeHidden) {
```

Comment:
I think that one reason is that when we're using redux in a connected component, we know that the store and the UI will always be in sync. Store changes cause a rerender (if it changes props we care about), and our UI is updated. In cases like this, someone might perform an action that results in a change to isDesignModeHidden, but we're not guaranteed to rerender this UI.